source/stylesheets/screen.css.scss

```diff
@@ -926,7 +926,7 @@ nav ul li ul li a {
   width: 70px;
 }
 
-@media only screen and (max-width: 870px) {
+@media only screen and (max-width: 1000px) {
```

Comment:
For all these places you're putting `1000px`, you can probably use `$tablet-width` :)